store/store.go

```diff
@@ -317,6 +317,7 @@ type PostStore interface {
 	GetDirectPostParentsForExportAfter(limit int, afterID string) ([]*model.DirectPostForExport, error)
 	SearchPostsInTeamForUser(paramsList []*model.SearchParams, userID, teamID string, page, perPage int) (*model.PostSearchResults, error)
 	GetOldestEntityCreationTime() (int64, error)
+	CheckIfAutoResponseByUserInChannelSince(options model.GetPostsSinceOptions, userId string) (bool, error)
```

Comment:
I see some other methods with a `(bool, error)` return signature starts with a `Has` prefix. I think that might be a better suited name for this. Also, `GetPostsSince` does not have `Channel` in its name.

I'll suggest to rename this to "HasAutoResponsePostByUserSince".